designs/dragonphy_top/qtm/analog_core.qtm.tcl

```diff
@@ -332,36 +346,17 @@ for {set idx 0} {$idx < 36} {incr idx} {
 
 ## Main ADCs
 
-set clk_adc_per [expr {$::env(constr_main_per) * $::env(constr_time_scale)}]
```

Comment:
we no longer need to calculate the delay of ADC outputs on a per-slice basis since they are all registered to the positive edge of `clk_adc`